httphandler/handlerequests/v1/datastructuremethods.go

```diff
@@ -86,4 +90,12 @@ func setTargetInScanInfo(scanRequest *utilsmetav1.PostScanRequest, scanInfo *cau
 		scanInfo.FrameworkScan = true
 		scanInfo.ScanAll = true
 	}
+
+	// Add "security" framework if not present
```

Comment:
What is the use-case?

Reply:
I will explain: As we spoke, kubescape receives the list of frameworks from the Operator. So why does KS need to handle this at all?

Reply:
@dwertent 
The requirement is to **always** scan the security framework when scanning framework(s). This function is the main junction all in-cluster scans go through so it is not only easier and faster to implement, but also making sure all scans requested will have the security framework appended.

Reply:
Correct, but the requirements do not specify how to implement...
By definition, Kubescape is a worker, meaning, kubescape carries out what was asked of him todo, without making "decisions" of what to run and how. For this reason, I expect the Operator/BE who triggers the scanning to handle such logic.

Reply:
This is a requirement for the worker. Not for the orchestrator.